pkg/sync/v12/data/SecurityContextConstraints.security.openshift.io/anyuid.yaml

```diff
@@ -17,6 +17,7 @@ metadata:
   annotations:
     kubernetes.io/description: anyuid provides all features of the restricted SCC
       but allows users to run with any UID and any GID.
+    openshift.io/reconcile-protect: "true"
```

Comment:
Without this annotation if the customer adds a user or group to the SCC, then it will get reverted in the next Sync pod run.

It's still marked as owned by Sync pod, but not reconciled.

Reply:
Removing after consultation. openshift.io/reconcile-protect: "true" should be applied by the customer when they decide to modify an SCC